lib/beaker-hostgenerator/data/vmpooler.rb

```diff
@@ -35,6 +35,14 @@ module Vmpooler
           'platform' => 'el-7-x86_64',
           'template' => 'centos-7-x86_64'
         },
+        'cisco5-64' => {
+          'platform' => 'cisco-5-x86_64',
+          'template' => 'cisco-nxos-9k-x86_64'
+        },
+        'cisco7-64' => {
```

Comment:
@joshcooper After speaking with @LuvCurves and @shermdog about this, there is quite a bit of uncertainty about whether it will be necessary to distinguish between the various hardware platforms. My preference is to be more explicit so I have updated the PR to add a hardware platform distinction to the beaker-hostgenerator spec string.